style.css

```diff
@@ -1,3 +1,126 @@
-#root {
-  color: red;
+/* ------- Mobile View --------- */
+
+body {
+  display: flex;
+  flex-direction: column;
+ 
+}
+
+header {
+  text-align: center;
+  background: conic-gradient(at 98% 10%, red 20%, yellow 60%, #1e90ff 100%);
+}
+#h1 {
+  font-size: xx-large;
+}
+
+#search_section {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  font-size: larger;
+}
+
+#number_of_episodes,
+h1 {
+  font-style: italic;
+  font-size: larger;
+  font-weight: bolder;
+  background-image: linear-gradient(to top, #0fd850 0%, #f9f047 100%);
+  border-radius: 25%;
+}
+
+h2 {
+  font-size: 1.5vw;
+  text-align: center;
+}
+main {
+  display: flex;
+  flex-direction: column;
+  border: solid;
+  align-items: center;
+  background: linear-gradient(
+      217deg,
+      rgba(255, 0, 0, 0.8),
+      rgba(255, 0, 0, 0) 0.71%
+    ),
+    linear-gradient(127deg, rgba(0, 255, 0, 0.8), rgba(0, 255, 0, 0) 70.71%),
+    linear-gradient(336deg, rgba(0, 0, 255, 0.8), rgba(0, 0, 255, 0) 120.71%);
+}
+
+.article_class {
```

Comment:
The styling on these cards is really sutle and quite simlar to the norm on cards, which I really like